distribution/openrct2.d.ts

```diff
@@ -4363,6 +4363,21 @@ declare global {
         textAlign: TextAlignment;
     }
 
+    interface ProgressBarWidget extends WidgetBase {
+        colour: number;
+        /**
+         * The percentage of the bar to fill, ranging from 0-100 (inclusive).
+         */
+        percentage: number;
+        /**
+         * Used to specify a range of values between which the bar will start blinking.
+         * Set both of these to the same value to disable blinking entirely.
+         * The range is inclusive.
+         */
```

Comment:
This mechanism is weird. Doesn't setting to -1 and 101 also mean there is no blinking? Anyways, I would suggest null values to hide the implementation details.

Reply:
You cannot set it to -1, it’s an unsigned integer.

Reply:
Makes sense. What about 0? Then an empty bar would blink, but you cannot see that :P
But again, I would prefer null values in both getter and setter.

Reply:
Makes sense. What about 0? Then an empty bar would blink, but you cannot see that :P
But again, I would prefer null values in both getter and setter.